go.mod

```diff
@@ -53,7 +53,6 @@ require (
 	github.com/heketi/heketi v10.3.0+incompatible
 	github.com/heketi/tests v0.0.0-20151005000721-f3775cbcefd6 // indirect
 	github.com/ishidawataru/sctp v0.0.0-20190723014705-7c296d48a2b5
-	github.com/json-iterator/go v1.1.11
```

Comment:
other than https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/staging/src/k8s.io/apimachinery/pkg/runtime/serializer/json/json.go
i cannot seem to find what else depends on refect2.
yet the main go.mod continues to have it.

is dropping reflect2 a goal too?

Reply:
yes, dropping reflect2 is a goal

we have transitive dependencies that use json-iterator and still pull in both of those deps
* k8s.io/kube-openapi (https://github.com/kubernetes/kube-openapi/pull/246 open to drop it)
* sigs.k8s.io/structured-merge-diff (https://github.com/kubernetes-sigs/structured-merge-diff/issues/202 open requesting moving away from it)
* go.etcd.io/etcd/client/v2
* github.com/prometheus/client_golang
* github.com/emicklei/go-restful